### DISCUSSION TITLE
[IntelliJ] Discarding suggestion is broken

### DISCUSSION THREAD

**Discussion by jakubtobiasz** [community/community#10506] [Copilot Conversations] [answered]
Hi!
After the latest update pressing [ESC] doesn't discard the suggestion anymore. Previously when I pressed [ESC] the suggestion was discard and I could write the code normally. It's a problem with both PHPStorm and Webstorm. I bet the all JetBrains IDEs are affected.
![2022-01-24 9 32 49 PM](https://user-images.githubusercontent.com/80641364/150860252-ff1db776-b2b8-489f-a7fa-eea809864d98.gif)

**Comment by jansorg** [comment, marked as answer]
@jakubtobiasz Thanks for reporting. Are you using a plugin like IdeaVim, for example?
Please add the info shown by `Help > About`.

**Reply by jakubtobiasz** [reply]
@jansorg Yes, I do. I'm using IdeaVim with Copilot and so far everything was fine. After the last updated I've noticed this strange behavior.

**Reply by jansorg** [reply]
This will be fixed with one of the next updates.
IdeaVim 1.9.3. changed a few things around Escape handling and Copilot added limited support for IdeaVim in December. Of of these changes may have caused the bug.
For the meantime, you could use an earlier version.

**Reply by danjessen** [reply]
I'm actually experiencing this issue again

**Reply by jakubtobiasz** [reply]
Newest version of PhpStorm, Copilot, IdeaVIM and no issue on my site :P

**Reply by danjessen** [reply]
Linux, Windows or MacOS ?

**Reply by jakubtobiasz** [reply]
macOS

**Comment by brokenthorn** [comment]
There is another issue with this. When I quickly add a new line between code using o and esc, a suggestion sometimes shows right after I exit into normal mode, and it won't let me move the cursor anymore until I press i to enter insert mode and then the suggestion disappears and then if I press esc, I can move the cursor again. So suggestions triggering in normal mode breaks Ideavim, by taking over the input while ideavim will not let the editor insert anything in normal mode.

![PowerToys_qkFvKqD3gR](https://github.com/community/community/assets/2119004/622e214f-aa43-4416-86f8-4932ade3d50f)
